### PR TITLE
feat(helm): support labels in values.yaml

### PR DESCRIPTION
**Pull request by razvan**
Part of: https://github.com/stackabletech/issues/issues/501

### CODE REVIEW

File: deploy/helm/secret-operator/templates/roles.yaml
Comment (Techassi) [inline deploy/helm/secret-operator/templates/roles.yaml:1]: note: Seems like we can include the labels in many more templated resources, like `SecretClass` and `StorageClass`.

File: deploy/helm/secret-operator/values.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@
 image:
   repository: docker.stackable.tech/stackable/secret-operator
   pullPolicy: IfNotPresent
+  pullSecrets: []
```
Comment (Techassi) [inline deploy/helm/secret-operator/values.yaml:6]: nit: Add a newline to separate these two blocks.

**Review by Techassi — CHANGES_REQUESTED** [review]

File: deploy/helm/secret-operator/templates/roles.yaml
Comment (razvan) [reply deploy/helm/secret-operator/templates/roles.yaml:1]: Shouldn't we?

Comment (razvan) [reply deploy/helm/secret-operator/templates/roles.yaml:1]: sorry. now I got what you meant. [Done](https://github.com/stackabletech/secret-operator/pull/352/commits/6a4c70067aa4cd347fa574cc42dff0c6b2bf20de)

**Review by Techassi — DISMISSED** [review]

**Review by Techassi — APPROVED** [review]

**Pull request merged**